api/envoy/extensions/geoip_providers/maxmind/v3/maxmind.proto

```diff
@@ -23,7 +23,13 @@ option (xds.annotations.v3.file_status).work_in_progress = true;
 // :ref:`anon_db_path <envoy_v3_api_field_extensions.geoip_providers.maxmind.v3.MaxMindConfig.anon_db_path>` must be configured.
 // [#extension: envoy.geoip_providers.maxmind]
 
+// [#next-free-field: 6]
 message MaxMindConfig {
+  // Full file path to the Maxmind country database, e.g. /etc/GeoLite2-Country.mmdb.
+  // Database file is expected to have .mmdb extension.
+  // Note that only one of country_db_path and city_db_path can be set.
```

Comment:
Add reference to proto fields, e.g. :ref:`city_db_path <envoy_v3_api_field_extensions.geoip_providers.maxmind.v3.MaxMindConfig.city_db_path>`